### PR TITLE
Document setting the prefix/suffix for the ruby collector

### PR DESCRIPTION
**Pull request by swebb**

### CODE REVIEW

File: pages/test_analytics/ruby_collectors.md.erb [hunk 1]
```diff
@@ -113,13 +124,15 @@ This would appear like so:
...
-```ruby
-Failure/Error: allow_any_instance_of(Object).to receive(:sleep)
-       Using `any_instance` to stub a method (sleep) that has been defined on a prepended module (Buildkite::TestCollector::Object::CustomObjectSleep) is not supported.
+```
+BUILDKITE_ANALYTICS_EXECUTION_NAME_PREFIX
+BUILDKITE_ANALYTICS_EXECUTION_NAME_SUFFIX
 ```
 
-you can fix them by being more specific in your stubbing, which is [better practice](https://relishapp.com/rspec/rspec-mocks/v/3-11/docs/working-with-legacy-code/any-instance), by replacing `allow_any_instance_of(Object).to receive(:sleep)` with `allow_any_instance_of(TheClassUnderTest).to receive(:sleep)`.
+When viewing a _test_ every _execution_ is displayed including it's corresponding prefix/suffix. For example:
```
Comment (olyism) [inline pages/test_analytics/ruby_collectors.md.erb:136]:
```suggestion
When viewing a _test_ every _execution_ is displayed including its corresponding prefix/suffix. For example:
```

Minor fix: its

**Comment by olyism** [comment]
Hi @swebb, thanks submitting the PR. I've added some minor suggestions above.

Secondly, I did a small tweak to the screenshot, what do you think about the following:

execution-prefix-suffix

In this screenshot I highlighted where the prefix and suffix are located. I also removed the top and bottom borders and zoomed in on the image so the screenshot's more visible to folks on smaller screens (e.g. mobile phone).

**Before**

Screen Shot 2022-10-03 at 9 31 04 pm

**After**

Screen Shot 2022-10-03 at 9 33 13 pm

**Review by olyism — CHANGES_REQUESTED** [review]
Looks great @swebb, requesting some small tweaks below.

**Comment by swebb** [comment]
Good suggestion @olyism. I've made all of those changes.

File: pages/test_analytics/ruby_collectors.md.erb [hunk 2]
```diff
@@ -113,13 +124,15 @@ This would appear like so:
...
-       Using `any_instance` to stub a method (sleep) that has been defined on a prepended module (Buildkite::TestCollector::Object::CustomObjectSleep) is not supported.
 ```
+BUILDKITE_ANALYTICS_EXECUTION_NAME_PREFIX
+BUILDKITE_ANALYTICS_EXECUTION_NAME_SUFFIX
+```
+
+When viewing a _test_ every _execution_ is displayed including its corresponding prefix/suffix. For example:
 
-you can fix them by being more specific in your stubbing, which is [better practice](https://relishapp.com/rspec/rspec-mocks/v/3-11/docs/working-with-legacy-code/any-instance), by replacing `allow_any_instance_of(Object).to receive(:sleep)` with `allow_any_instance_of(TheClassUnderTest).to receive(:sleep)`.
+<%= image "execution-prefix-suffix.png", width: 2086/2, height: 294/2, alt: "Screenshot of test executions including a prefix and suffix." %>
```
Comment (olyism) [inline pages/test_analytics/ruby_collectors.md.erb:138]:
```suggestion
<%= image "execution-prefix-suffix.png", width: 2048/2, height: 400/2, alt: "Screenshot of test executions including a prefix and suffix." %>
```
Hi @swebb just one last minor thing: if we could fix the image dimensions I think we're ready to go.

**Review by olyism — CHANGES_REQUESTED** [review]

File: pages/test_analytics/ruby_collectors.md.erb (hunk 2 above)
Comment (swebb) [reply pages/test_analytics/ruby_collectors.md.erb]: Thanks @olyism.

**Review by olyism — APPROVED** [review]
Thanks @swebb LGTM 🚀

**Pull request merged**